app.js

```diff
@@ -96,12 +96,19 @@ Test this function by hand in the console to get it working, and when you think
 var testArray = [2,3,4]; //eslint-disable-line
 
 function sumArray(testArray){ //eslint-disable-line
+  var calculatedSum = sum(testArray[0], parseInt(sum(testArray[1],testArray[2])[0]))[0];
```

Comment:
This does make the test pass, but what if you use a `testArray` with more than three elements?